apps/admin/backend/schema.sql

```diff
@@ -42,13 +42,23 @@ create table cvrs (
   ballot_type text not null 
     check (ballot_type = 'absentee' or ballot_type = 'precinct' or ballot_type = 'provisional'),
   batch_id text not null,
-  batch_label text not null,
   party_id text,
   precinct_id text not null,
   scanner_id text not null,
   sheet_number integer check (sheet_number is null or sheet_number > 0),
   votes text not null,
   created_at timestamp not null default current_timestamp,
+  foreign key (election_id) references elections(id)
+    on delete cascade,
+  foreign key (batch_id, scanner_id) references scanner_batches(batch_id, scanner_id)
+);
+
+create table scanner_batches (
+  batch_id text not null,
+  label text not null,
+  scanner_id text not null,
+  election_id varchar(36) not null,
+  primary key (batch_id, scanner_id),
```

Comment:
For my education: is this because batch IDs are only unique across batches from a single scanner?

(If that's the case, I guess that cancels out my comment about moving `scanner_id` out of `cvrs`, since we'd need it to join `scanner_batches` anyway)

Reply:
In the real world, batch IDs should be unique. They are UUID's. I'm never quite sure about this point - do I assume UUIDs are, indeed unique? That the probability of collision is so low I shouldn't build around it? Because if that's the case, then we could have have simple batch_id here.

I also had our fixtures in mind, which do *not* have unique batch ID's, but I can fix that.

Reply:
Oh yeah, if we're using UUIDs, I believe we'll be fine even when combining data across multiple machines